#goal for the manipulation action
manipulation/ManipulationPlanRequest[] action_plan
---
#result 
moveit_msgs/MoveItErrorCodes error_code
---
#feedback
manipulation/ManipulationPlanRequest current_action